pkg/resource/plugin/analyzer_plugin.go

```diff
@@ -311,6 +329,42 @@ func (a *analyzer) GetPluginInfo() (workspace.PluginInfo, error) {
 	}, nil
 }
 
+func (a *analyzer) Configure(policyConfig map[string]AnalyzerPolicyConfig) error {
+	label := fmt.Sprintf("%s.Configure(...)", a.label())
+	logging.V(7).Infof("%s executing", label)
+
+	c := make(map[string]*pulumirpc.PolicyConfig)
+
+	for k, v := range policyConfig {
+		var el pulumirpc.EnforcementLevel
```

Comment:
We are defaulting into the zero value - is that right?  Should we be explicit about what the default is here?

Reply:
Changed it to require a valid enforcement level, rather than defaulting like this. In practice, all current callers are going to pass valid enforcement levels given how we build-up the config.